### ISSUE TITLE
Screenreader change how assignment results are read out loud

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5598]
**Current behavior**

<img width="719" height="301" alt="Image" src="https://github.com/user-attachments/assets/633f8cc3-49b5-4120-92e5-22feb779c85f" />

How the table is spoken out loud (not verbatim but close):
'third cell yes, fourth cell no, fifth cell Abstain. one Klara Ziegel rot 42 percent 6 21percent 3 28 percent 4 2 Tim Ziegel....

**Wanted behavior**
One line should be spoken as one unit 
"Poll Options [the options]. Read out loud in the order they placed"
"Name, structure level: the structure level. Yes: 6 votes 42%. No:  3 votes 21%. Abstain: 4 votes 28%"
"Name, ...."
Please use the actual percentages not the abbreviations.

This Issue will be probably need some more loops if worked on

**Reproduction:**
1. activate vote delegations in the setting (participant-detail -:> delegate setting)
2. set yourself to present
3. delegate the vote from the opther participant to yourself (this make the next steps easier)
4. create an election with two or more candidates
5. create a poll where you and the other participants can partace in (I would recommend a Y/N/A poll but the others should work too)
6. cast your votes
7. end and publish or just end the poll
8. now the results should be read as described
